README.md

```diff
@@ -38,9 +38,51 @@ For more information about the default configuration, see [Default Secure Landin
 | -------------------------------- | -------------------------------- | ---------------------------------- |
 | ![VSI](./.docs/images/vsi.png)   | ![ROKS](./.docs/images/roks.png) | ![Mixed](./.docs/images/mixed.png) |
 
-## Customizing your environment
+## Before you begin
 
-You can customize your environment with Secure Landing Zone in two ways: by using Terraform input variables and by using the `override.json` file.
+Complete the following steps before you deploy the Secure Landing Zone module.
+### Set up an IBM Cloud Account
+
+1.  Make sure that you have an IBM Cloud Pay-As-You-Go or Subscription account:
+
+    - If you don't have an IBM Cloud account, [create one](https://cloud.ibm.com/docs/account?topic=account-account-getting-started).
+    - If you have a Trial or Lite account, [upgrade your account](https://cloud.ibm.com/docs/account?topic=account-upgrading-account).
+
+### Configure your IBM Cloud account for Secure Landing Zone
+
+1.  Log in to [IBM Cloud](https://cloud.ibm.com) with the IBMid you used to set up the account. This IBMid user is the account owner and has full IAM access.
+1.  [Complete the company profile](https://cloud.ibm.com/docs/account?topic=account-contact-info) and contact information for the account. This profile is required to stay in compliance with IBM Cloud Financial Service profile.
+1.  [Enable the Financial Services Validated option](https://cloud.ibm.com/docs/account?topic=account-enabling-fs-validated) for your account.
+1.  Enable virtual routing and forwarding (VRF) and service endpoints by creating a support case. Follow the instructions in  [enabling VRF and service endpoints](https://cloud.ibm.com/docs/account?topic=account-vrf-service-endpoint&interface=ui#vrf).
+
+### Set up account access (Cloud IAM)
+
+1.  If you are provisioning resources manually, create an IBM Cloud [API key](https://cloud.ibm.com/docs/account?topic=account-userapikey#create_user_key). The user who owns this key must have the Administrator role.
```

Comment:
@vburckhardt: Can we be more specific about what `provisioning resources manually` means? Is it outside of the catalog through Terraform directly or something else?

Reply:
I removed the sentence, it did not fully make sense in first place